extensions/poetry/main.ts

```diff
@@ -0,0 +1,53 @@
+import { PhylumApi } from "phylum"
+import { parse } from "https://deno.land/std/flags/mod.ts"
```

Comment:
This was not intentional, but I don't have a strong opinion over which option is better.

I see Deno's std is in constant flux; pinning the version would allow for less maintenance but it would expose us to vulnerabilities/bugs, using latest on the other hand would require us to regularly run and check the extensions for breaking changes.